pkg/request/request.go

```diff
@@ -27,6 +27,27 @@ func DeleteRequest(url string, dryRun bool, connection *sdk.Connection) error {
 	return nil
 }
 
+func PatchRequest(url string, body []byte, dryRun bool, connection *sdk.Connection) error {
+	request := connection.Patch()
+	err := ApplyPathArg(request, url)
+	if err != nil {
+		return fmt.Errorf("can't parse url '%s': %v\n", url, err)
+	}
+	request.Bytes(body)
+	if dryRun {
+		fmt.Printf("DRYRUN: Would have called %v.\n", request.GetPath())
```

Comment:
It would be nice to include the action (in this case Patch)

Reply:
Same for the DeleteRequest method above

Reply:
Added operation name in the message